cara menghitung anak tangga()

RiskaMahardika13 opened the pull request:
nama:I kadek riska Mahardika
nim:2313231036
kita membuat code menghitung anak tangga code dibagi menjadi 4

ini code ke 2

riskamahardika12 committed:
cara menhitung anak tangga()

nama:i kadek riska mahardika
nim:231323036
kita membuat code menghitung anak tangga code dibagi menjadi 4

ini code ke 2

riskamahardika12 committed:
cara menghitung anak tangga()

nama:I kadek riska mahardika
nim:2313231036
kita membuat code menghitung anak tangga code dibagi menjadi 4

ini code ke 2

kyo21 reviewed (changes requested):
Kontribusinya kosong, ga ada tambahan kode

riskamahardika12 committed:
cara menghitung anak tangga()

nama:I kadek riska mahardika
nim:2313231036
kita membuat code menghitung anak tangga code dibagi menjadi 4

menambahkan void

riskamahardika12 committed:
Merge branch 'main' of https://github.com/RiskaMahardika13/Anak-Tangga

kyo21 reviewed (approved):
Accepted
